### PR TITLE
Cabin crew, take seats for landing!

### PR DESCRIPTION
**Pull request by baulig**

### CODE REVIEW

File: mcs/class/Mono.Security/Mono.Security.X509/X509Store.cs [hunk 1]
```diff
@@ -279,7 +334,7 @@ private X509Certificate LoadCertificate (string filename)
 		{
 			byte[] data = Load (filename);
 			X509Certificate cert = new X509Certificate (data);
-#if !MOBILE
+#if !NET_2_1
```
Comment (marek-safar) [inline mcs/class/Mono.Security/Mono.Security.X509/X509Store.cs]: We not longer use NET_2_1 define in sources, please revert your change

File: mcs/class/Mono.Security/Mono.Security.X509/X509Store.cs [hunk 2]
```diff
@@ -371,7 +426,7 @@ private ArrayList BuildCrlsCollection (string storeName)
 			}
 			return list;
 		}
-#if !MOBILE
+#if !NET_2_1
```
Comment (marek-safar) [inline mcs/class/Mono.Security/Mono.Security.X509/X509Store.cs]: We not longer use NET_2_1 define in sources, please revert your change

File: mcs/tools/btls/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+[assembly: SatelliteContractVersion (Consts.FxVersion)]
+[assembly: AssemblyInformationalVersion (Consts.FxFileVersion)]
+
+[assembly: CLSCompliant (true)]
+[assembly: NeutralResourcesLanguage ("en-US")]
+
+[assembly: ComVisible (false)]
+
+[assembly: AssemblyDelaySign (true)]
+[assembly: AssemblyKeyFile ("../../class/mono.pub")]
```
Comment (marek-safar) [inline mcs/tools/btls/AssemblyInfo.cs]: Do you really have to sign this tool?

Comment (baulig) [inline mcs/tools/btls/AssemblyInfo.cs]: Yes.  For the moment, I would rather use the internal API in here and have internals visible than depending on `Mono.Btls.Interface`.
At least until we have figured out whether we're going to install `Mono.Btls.Interface` and how that's going to work with the test suite.

**Comment by baulig** [comment]
Ok, I just finished some cleanups, now let's see whether Jenkins can build this.

**Comment by baulig** [comment]
build

**Comment by JustArchi** [comment]
Any clue why thing like this happens? http://hastebin.com/jijayilisi.js

**Comment by vargaz** [comment]
This can be closed right ?

**Pull request closed without merging**